content/source/docs/enterprise/admin/upgrades.html.md

```diff
@@ -10,13 +10,38 @@ version. [Learn more about availability during upgrades here](../system-overview
 
 ## Online
 
+### Replicated Administration Console
```

Comment:
[Elsewhere in the docs](https://www.terraform.io/docs/enterprise/install/installer.html#tls-key-amp-cert) we refer to this as the "Replicated Console"

```suggestion
### Replicated Console
```